packages/block-editor/src/components/block-preview/auto.js

```diff
@@ -91,6 +91,9 @@ function ScaledBlockPreview( {
 					documentElement.style.width = '100%';
 					bodyElement.style.padding = __experimentalPadding + 'px';
 
+					// Necessary for proper previews of styles with border styles assigned to the body.
+					bodyElement.style.border = 'none';
```

Comment:
It's interesting that some styling is set using JavaScript here and some is set using CSS on line 52 above. Did you consider putting `border: none` into the CSS above? What's the better approach?

Reply:
Thanks for the CR, @noisysocks :)

I initially chose that place, so that I can add some inline docs explaining the reason behind removing the border. Maybe line 52 is cleaner, I've just moved it there.

Reply:
Much of a muchness I suppose, thanks for considering 😀